service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -67,50 +75,35 @@ public ResponseEntity<RunSetStateResponse> postRunSet(RunSetRequest request) {
         InputGenerator.buildInputs(request.getWorkflowParamDefinitions(), entityResponse);
 
     // Submit the workflow and get its ID:
-    WorkflowIdAndStatus workflowResponse;
+    RunId workflowResponse;
     try {
       workflowResponse = submitWorkflow(request.getWorkflowUrl(), params);
     } catch (cromwell.client.ApiException e) {
       log.warn("Cromwell submission failed. ApiException", e);
       return new ResponseEntity<>(HttpStatus.valueOf(e.getCode()));
     } catch (JsonProcessingException e) {
-      // Should be super rare than jackson cannot convert an object to Json...
+      // Should be super rare that jackson cannot convert an object to Json...
       log.warn("Failed to convert inputs object to JSON", e);
       return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
     }
 
+    //    RunStateResponse state = new RunStateResponse();
+
     // Return the result:
     return new ResponseEntity<>(
         new RunSetStateResponse()
             .runSetId(UUID.randomUUID().toString())
             .addRunsItem(
-                new RunStateResponse()
-                    .runId(workflowResponse.getId())
-                    .state(cromwellToCbasRunStatus(workflowResponse.getStatus())))
+                new RunStateResponse().runId(workflowResponse.getRunId()).state(RunState.RUNNING))
```

Comment:
To Reviwers: Here, I'm hard-coding the state of the individual workflow to `RUNNING`, when we want to fetching the state that Wes gives us. `workflowResponse` is of type `RunId`, and so I'm struggling with how to effectively use the right types to get the state from the given workflow.

Reply:
Maybe "UNKNOWN" (if there is one)?

Reply:
Should I be hard-coding this in though?